blocks/subquery/edit.tsx

```diff
@@ -142,12 +140,23 @@ export default function Edit({
     currentPostId,
   })}&${termQueryArgs}`;
 
+  useEffect(() => {
+    if (!isFirstPost) {
+      return;
+    }
+
+    if (postTypes?.length > 0) {
+      return;
+    }
+
+    setAttributes({ postTypes: allowedPostTypes.map((type) => type.slug) });
```

Comment:
Could use destructuring here:
```suggestion
    setAttributes({ postTypes: allowedPostTypes.map(({ slug }) => slug) });
```